package.json

```diff
@@ -7,23 +7,23 @@
   "engines": {
     "node": ">=16"
   },
-  "packageManager": "yarn@4.1.1",
+  "packageManager": "yarn@4.2.2",
```

Comment:
was about to PR only this Yarn bump ^ from running `yarn` on `master`
remaining dep bumps lgtm

as an aside (on corepack 0.25.2 locally)

![image](https://github.com/endojs/endo/assets/1881059/7c04c226-4329-439e-aa39-024a9535e1e2)

it's not obvious why `corepack enable` isn't enabling like suggested

![image](https://github.com/endojs/endo/assets/1881059/9c106e5a-1722-479a-852c-4264a61816db)

wonder if it's worth mentioning somewhere in docs something like _you may need to set your global yarn version to v4 to run corepack`

Reply:
You shouldn't have to change your global yarn version. Once corepack is enabled it ignores the global version.

The "Presence…" message you're seeing above is because corepack isn't enabled. It says "Corepack must currently be enabled by running `corepack enable` in your terminal`. I don't know why your command didn't achieve that but maybe try running it separately.

```
corepack enable
which yarn
```

Reply:
repro round 2

<img width="1021" alt="Screenshot 2024-05-24 at 4 42 18 PM" src="https://github.com/endojs/endo/assets/1881059/6803ecd2-7b0b-4305-8785-cc19ad66c46e">

but agree, shouldn't have to change the global yarn version, like i have to at the root level as a workaround, `corepack enable` should just work 🤔